DesignSafe-CI/portal
# task/DES-2179, task/DES-2141: Remove data depot footer from workspace; Update NHERI logo

## Overview: ##
Updates the NHERI logo and removes the footer from frontend app pages. (i.e. only included on cms pages)

## PR Status: ##


## Related Jira tickets: ##

* [DES-2179](https://jira.tacc.utexas.edu/browse/DES-2179)
* [DES-2141](https://jira.tacc.utexas.edu/browse/DES-2141)

## Testing Steps: ##
1. Go to designsafe.dev and confirm footer **is** present
2. Confirm logo is new updated logo, and not just "NHERI" in h1 text format
3. Go to https://designsafe.dev/data/browser/ or any other frontend app and confirm footer **is not** present

## UI Photos:
<img width="1792" alt="Screen Shot 2022-02-03 at 4 19 27 PM" src="https://user-images.githubusercontent.com/20326896/152438735-6116598a-9977-48af-8619-d23b24020d12.png">
